### PR TITLE
optionally disable inspect style

### PR DESCRIPTION
**Pull request by tcql**
Lets you leave your map with it's own style if you want to just have a hovering inspect popup automatically without the x-ray-esque style

### CODE REVIEW

File: lib/MapboxInspect.js [hunk 1]
```diff
@@ -74,8 +74,9 @@ function MapboxInspect(options) {
     buildInspectStyle: stylegen.generateInspectStyle,
     renderPopup: renderPopup,
     popup: popup,
-    selectThreshold: 5
```
Comment (tcql) [inline lib/MapboxInspect.js]: looks like the previous PR merges didn't include a `,` here

Comment (lukasmartinelli) [inline lib/MapboxInspect.js]: Fixed it.

**Comment by lukasmartinelli** [comment]
Such good ideas :) Thanks.

**Comment by lukasmartinelli** [comment]
It could be that you were already able to do that before?

http://mapbox-gl-inspect.lukasmartinelli.ch/examples/inspect-only.html#3/40.00/-74.50

```
map.addControl(new MapboxInspect({
  showInspectMap: true,
  showInspectButton: false
}));
```

Do I understand correctly this option is when you want to enable popup only when the inspect mode is toggled but then show no inspect style.

While the example above just allows the popup on the normal map.

**Pull request merged**